src/runtime/hashmap.go

```diff
@@ -413,15 +418,15 @@ func hashmapBinaryGet(m *hashmap, key, value unsafe.Pointer, valueSize uintptr)
 		memzero(value, uintptr(valueSize))
 		return false
 	}
-	hash := hash32(key, uintptr(m.keySize))
+	hash := hash32(key, uintptr(m.keySize), m.seed)
 	return hashmapGet(m, key, value, valueSize, hash)
 }
 
 func hashmapBinaryDelete(m *hashmap, key unsafe.Pointer) {
 	if m == nil {
 		return
 	}
```

Comment:
Now that all the `hashmap*Set`, `hashmap*Get`, and `hashmap*Delete` functions have nil checks themselves, I think the nil checks can be removed from the functions `hashmapSet`, `hashmapGet`, and `hashmapDelete`.